### PR TITLE
Wishlist Page

### PR DESCRIPTION
**Pull request by bajajcodes**
- Wishlist get all of your favourite items at one place.
- Get rid of items or move to cart in one click and meanwhile enjoy responsive attitude.

https://user-images.githubusercontent.com/29247011/154841621-02e5243b-57b6-4edf-8078-e9ad58da54d9.mp4

### CODE REVIEW

File: css/products.css [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    padding-bottom: var(--size-13);
+}
+
+.aside-navbar.show {
+    transform: translateX(0);
+}
+
+.beautify-aside {
+    z-index: 5;
+    background-color: rgba(255, 253, 253, 0.986);
```
Comment (RakshithBhat03) [inline css/products.css]: You can use a variable here and throughout the code

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+<body>
+
+    <header class="header">
+
+        <nav class="navbar">
+
+            <div class="navbar-brand">
+                <a href="/" class="nav-link">
+                    <img src="../assets/logo/logo.svg" alt="" width="40" height="40"
```
Comment (RakshithBhat03) [inline pages/wishlist.html]: add more descriptive `alt`

**Review by RakshithBhat03 — APPROVED** [review]
Suggested minor changes. Else everything looks good.

File: css/products.css (hunk 1 above)
Comment (bajajcodes) [reply css/products.css]: I have implemented the changes.

File: pages/wishlist.html (hunk 2 above)
Comment (bajajcodes) [reply pages/wishlist.html]: I have added description in `alt` attribute, thanks for pointing out.

File: css/products.css [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+/* main-content cards */
+.cards-container {
+    margin: 2em;
+    display: grid;
+    row-gap: var(--size-11);
+    grid-template-columns: repeat(auto-fill,minmax(380px, 1fr));
+  }
+
+.cards-container .card{
+    width: 380px;
```
Comment (Shivanshsharma01) [inline css/products.css]: can use rems here

File: css/products.css [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+}
+
+.cards-container .card:hover{
+    box-shadow: var(--size-3) var(--size-3) 0 #000;
+    transform: scale(1.1);
+    z-index: 4;
+}
+
+.card-img{
+    height: 300px;
```
Comment (Shivanshsharma01) [inline css/products.css]: can use relative units as a good practice

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean with good use of variables. Can implement suggested changes

File: css/products.css (hunk 3 above)
Comment (bajajcodes) [reply css/products.css]: I have implemented the suggested change :)

File: css/products.css (hunk 4 above)
Comment (bajajcodes) [reply css/products.css]: I have implemented the suggested change :)

**Pull request merged**